ansible_collections/studyly/git_mactp/plugins/module_utils/git_actions.py

```diff
@@ -224,12 +273,9 @@ def push(self):
         Push changes to remote repo.
 
         args:
-            * path:
-                type: path
-                description: git repo local path.
-            * cmd_push:
-                type: list()
-                description: list of commands to perform git push operation.
```

Comment:
Were these obsolete already? The code below shows no signs of removal of related stuff.

Maybe you could add docs about the actual valid params here?